source/Analysis/include/AnalyzerGeometry.h

```diff
@@ -37,6 +37,16 @@ class TProfile;
 class Ring;
 class RootWTable;
 
+
+// Formatting parameters
+static const int coordPrecision = 3;
+static const int zOverlapPrecision = 5;
+static const int anglePrecision = 1;
+// Some strings for the html formatting
+static const std::string subStart   = "<sub>";      // These only should be needed
+static const std::string subEnd     = "</sub>";
+
```

Comment:
So, for subStart and subEnd, you may directly use definition from global_constants.h :-)